docs/examples/metrics/event-consumers.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2020 Google LLC
```

Comment:
There is only one event consumer, should this file be named `event-consumer.yaml` instead?